### DISCUSSION TITLE
Small form factor headset screw hole position

### DISCUSSION THREAD

**Discussion by 70nmr** [sn-lab/MouseGoggles#21] [Q&A] [answered]
Hello,
I'm trying to assemble a small form factor headset, but the screw hole locations on the PCB don't seem to match the locations on the 3D printed headset.

I used those files.

Hardware/3D printing/miniHeadet_V1.stl
Hardware/PCB/RaspyGC9307_minibrdV1_2023-07-01.zip

Are there any other versions of these files?

**Comment by misaacson01** [comment, marked as answer]
Hello,

Yes you're correct, that PCB file is for a newer version than the 3D print (the small form factor design is still in flux, which is why it doesn't have assembly instructions yet), sorry for the mistake! If you've already ordered all these parts, I suppose the easier option is to get new 3D prints to match the PCB since the PCB takes longer to order, so I'll focus on that. 

Newer versions of the 3D prints (which match the dimensions of the PCB you have) have individual eyepieces that are separately screwed to the PCB to allow some adjustment to inter-eye distance. I've just added these new parts to the 3D prints folder (miniHeadset_V2_eyepiece.stl and miniHeadset_V2_backing.stl).

Just a few notes on assembling this version, focusing on differences with the main version described in the repo: 
- You'll need to print 2 eyepieces for the mini headset. If you want them to be perfectly symmetric, one eyepiece should be mirror-flipped. This isn't functionally necessary, but without flipping, one eyepiece will have the lens slot on the top and the other on the bottom.
- There are two different PCBs that need to be printed for the mini headset -- one that attaches to the headset, and another to the Raspberry Pi. In addition to the ribbon cable connectors, these PCBs uses omnetics SPI connectors to attach the 2 boards, which need to be specifically assembled following the orientation below (male pins on the top row on the Raspberry Pi PCB; male pins on the bottom row on the headset PCB): ![(left) Assembled Raspberry Pi PCB. (right) Assembled headset PCB](https://raw.githubusercontent.com/sn-lab/MouseGoggles/main/Images/MiniHeadsetPCBsAssembled.png)

**Reply by 70nmr** [reply]
Thank you for this generous information. Thank you so much.
